internal/services/noop_reservation_service_test.go

```diff
@@ -0,0 +1,29 @@
+package services
+
+import (
+	"context"
+	"testing"
+
+	_ "github.com/RHEnVision/provisioning-backend/internal/testing/initialization"
```

Comment:
This import needs to be added in most our tests for a consistent experience (no nil pointer exceptions, all tests do logging in human-readable form).